portfolio/src/main/webapp/index.html

```diff
@@ -171,12 +171,22 @@ <h2 class="mb-5">Interests</h2>
     <hr class="m-0">
     </div>
     <!-- Servlet-->
-    <section class="resume-section" id="servlet">
+    <section class="resume-section" id="comments">
+      
       <div class="resume-section-content">
-        <h2 class="mb-5">Servlet Practice</h2>
-        <p>Click here to see a message:</p>
-        <button onclick="getMessage()">Click me!</button>
-        <div id="message-container"></div>
+
+        <h2 class="mb-5">Leave a comment</h2>
+        <form action="/new-comment" method="POST">
+            <input type="text" name="user-name" placeholder="Name..."/>
```

Comment:
The closing / can be omitted for the input tag.

Reply:
Done, thanks